.github/workflows/ghcr-image-build-and-publish.yml

```diff
@@ -54,6 +60,7 @@ jobs:
         uses: docker/build-push-action@v3.2.0
         with:
           context: .
+          platforms: linux/amd64,linux/arm64
           push: ${{ github.event_name != 'pull_request' }}
```

Comment:
> on:
>  push:

Could you add `pull_request` so that this can be tested on the CI

Reply:
done